curriculum/challenges/english/blocks/workshop-html-video-player/68f27139bddf9f6bf2821acf.md

```diff
@@ -0,0 +1,66 @@
+---
+id: 68f27139bddf9f6bf2821acf
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+
+Let's add one final `source` element. This one will use the `video/quicktime` MIME type.
```

Comment:
you use "let's" often, remember that it stans for "Let us", and you should not use first person in the text